Arrangement_on_surface_2/include/CGAL/Arr_tracing_traits_2.h

```diff
@@ -35,206 +30,207 @@
 
   bool intersect_op() const
-  { return (0 != (m_flags & (0x1 << INTERSECT_OP))); }
+  { return (0 != (m_flags & (0x1 << INTERSECT_2_OP))); }
 
   bool are_mergeable_op() const
-  { return (0 != (m_flags & (0x1 << ARE_MERGEABLE_OP))); }
+  { return (0 != (m_flags & (0x1 << ARE_MERGEABLE_2_OP))); }
 
   bool merge_op() const
-  { return (0 != (m_flags & (0x1 << MERGE_OP))); }
+  { return (0 != (m_flags & (0x1 << MERGE_2_OP))); }
 
   bool construct_opposite_op() const
-  { return (0 != (m_flags & (0x1 << CONSTRUCT_OPPOSITE_OP))); }
+  { return (0 != (m_flags & (0x1 << CONSTRUCT_2_OPPOSITE_2_OP))); }
 
   bool compare_endpoints_xy_op() const
-  { return (0 != (m_flags & (0x1 << COMPARE_ENDPOINTS_XY_OP))); }
+  { return (0 != (m_flags & (0x1 << COMPARE_ENDPOINTS_XY_2_OP))); }
+
+  bool approximate_op() const
+  { return (0 != (m_flags & (0x1 << APPROXIMATE_2_OP))); }
 
   // left-right
 
   bool parameter_space_in_x_op() const
-  { return (0 != (m_flags & (0x1 << PARAMETER_SPACE_IN_X_OP))); }
+  { return (0 != (m_flags & (0x1 << PARAMETER_SPACE_IN_X_2_OP))); }
 
   bool is_on_x_identification_op() const
-  { return m_flags & (0x1 << IS_ON_X_IDENTIFICATION_OP); }
+  { return m_flags & (0x1 << IS_ON_X_IDENTIFICATION_2_OP); }
 
   bool compare_y_on_boundary_op() const
-  { return (0 != (m_flags & (0x1 << COMPARE_Y_ON_BOUNDARY_OP))); }
+  { return (0 != (m_flags & (0x1 << COMPARE_Y_ON_BOUNDARY_2_OP))); }
 
   bool compare_y_near_boundary_op() const
-  { return m_flags & (0x1 << COMPARE_Y_NEAR_BOUNDARY_OP); }
+  { return m_flags & (0x1 << COMPARE_Y_NEAR_BOUNDARY_2_OP); }
 
   // bottom-top
 
   bool parameter_space_in_y_op() const
-  { return (0 != (m_flags & (0x1 << PARAMETER_SPACE_IN_Y_OP))); }
+  { return (0 != (m_flags & (0x1 << PARAMETER_SPACE_IN_Y_2_OP))); }
 
   bool is_on_y_identification_op() const
-  { return m_flags & (0x1 << IS_ON_Y_IDENTIFICATION_OP); }
+  { return m_flags & (0x1 << IS_ON_Y_IDENTIFICATION_2_OP); }
 
   bool compare_x_on_boundary_op() const
-  { return (0 != (m_flags & (0x1 << COMPARE_X_ON_BOUNDARY_OP))); }
+  { return (0 != (m_flags & (0x1 << COMPARE_X_ON_BOUNDARY_2_OP))); }
 
   bool compare_x_near_boundary_op() const
-  { return m_flags & (0x1 << COMPARE_X_NEAR_BOUNDARY_OP); }
+  { return m_flags & (0x1 << COMPARE_X_NEAR_BOUNDARY_2_OP); }
 
 public:
-  /*! Construct default */
+  /*! constructs default. */
   template<typename ... Args>
-  Arr_tracing_traits_2(Args ... args) :
-    Base(args...)
-  {
-    enable_all_traces();
-  }
+  Arr_tracing_traits_2(Args ... args) : Base(std::forward<Args>(args)...)
+  { enable_all_traces(); }
 
-  /*! Disable copy constructor.
-   */
+  /*! disables copy constructor. */
   Arr_tracing_traits_2(const Arr_tracing_traits_2&) = delete;
 
-  /*! Enable the trace of a traits operation
-   * \param id the operation identifier
+  /*! enables the trace of a traits operation.
+   * \param id the operation identifier.
    */
   void enable_trace(Operation_id id) { m_flags |= 0x1 << id; }
```

Comment:
`89>C:\CGAL_ROOT\CGAL-6.1-Ic-19\include\CGAL\Arr_tracing_traits_2.h(179,55): warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?) [C:\CGAL_ROOT\CGAL-6.1-Ic-19\cmake\platforms\MSVC-2022-Community-Release\test\Arrangement_on_surface_2_Examples\count_and_trace.vcxproj]` not sure if it's a new warning